gengokit/gengokit.go

```diff
@@ -56,3 +58,25 @@ func NewTemplateExecutor(sd *svcdef.Svcdef, conf Config) (*TemplateExecutor, err
 		FuncMap:      funcMap,
 	}, nil
 }
+
+// ApplyTemplate applies the passed template with the Executor
```

Comment:
This documentation comment doesn't really tell me very much. If this were called `Render`, I could document it like, "Render returns a template filled with the Executor field values."

This is another case where `Executor` seems like a weird name. And this rendering function being a method on a set of template values strikes me as a little weird. Like, this would make the most sense as a function that takes a template reference and a set of values that should be available to the template.

That's a nitpick, but as it is, it seems like functionality bolted onto a somewhat strange location.